control-plane/config/200-controller/200-addressable-resolver-cluster-role.yaml

```
@@ -0,0 +1,42 @@
+# Copyright 2021 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: kafka-addressable-resolver
```

Since this is a `ClusterRole` and it could conflict with other unrelated projects, I'd recommend adding `knative-` as prefix
```suggestion
  name: knative-kafka-addressable-resolver
```

---

I noticed we don't do this in the other resources but we should.

---

+1 Done